docs/en/sql-reference/statements/select/except.md

```diff
@@ -30,7 +30,7 @@ Here is a simple example that returns the numbers 1 to 10 that are _not_ a part
 Query:
 
 ``` sql
-SELECT number FROM numbers(1,10) EXCEPT SELECT number FROM numbers(3,6);
+SELECT number FROM numbers(1,10) EXCEPT SELECT number FROM numbers(3,8);
```

Comment:
Why?

Reply:
Oh, it's my fault! I misunderstood the usage of numbers functions. I have already checked the usage of the numbers function. Now I understand. Thank you for your answer.